frontend/src/components/TravelerGroup/TravelerGroupCard/TravelerGroupCard.tsx

```diff
@@ -71,10 +71,11 @@ export default function TravelerGroupCard(props: TravelerCardProps) {
         <div>
           {props.travelerGroup.travelerList.map((traveler) => {
             return (
-              <div key={traveler.id}>
-                <h6>{traveler.name}</h6>
-                <p>{traveler.amount}€</p>
-              </div>
+              <ul key={traveler.id}>
+                <li>
+                  {traveler.name} {traveler.amount}€
+                </li>
+              </ul>
```

Comment:
Are you sure you want to have an unordered List for every traveller? Or do you want one list with all travellers inside? 🤔